GDJS/Runtime/types/project-data.d.ts

```diff
@@ -66,6 +75,8 @@ declare type BasicObjectNetworkSyncData = {
   pfx: number;
   /** Permanent force on Y */
   pfy: number;
+  /* name :*/
+  n: string;
```

Comment:
let's make it so it can be undefined: `n?: string` (so that we don't pass it if multiplayer)